src/Configuration/Config.Azure.KeyVault.Secrets/ref/Microsoft.Extensions.Configuration.Azure.KeyVault.Secrets.Manual.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
@BrennanConroy reminder that we should do API review for all of these, preferably on Monday because of the tight schedule.